src/shared/helpers/sendZoomRequest.ts

```diff
@@ -43,48 +39,20 @@ const sendZoomRequest = async (
     zoomAPIConfig,
   } = opts;
 
-  // Check if our host is the Harvard HUIT Apigee system
-  const usingHarvardApigee = (
-    zoomAPIConfig.zoomHost
-    && zoomAPIConfig.zoomHost.indexOf(HARVARD_APIGEE_HOSTNAME_SUFFIX) >= 0
-  );
-
   /* ------------------ Make sure we have a token ----------------- */
 
-  let token;
-  if ((opts as any).zoomAPIConfig.token) {
-    // Use existing token
-    ({ token } = (opts as any));
-  } else {
-    // Generate a token
-    token = jwt.sign(
-      {
-        iss: (opts as any).key,
-        exp: (Date.now() + (EXPIRY_SEC * 1000)),
-      },
-      (opts as any).secret,
-    );
-  }
+  const token = await getCurrentAccessToken(zoomAPIConfig);
 
   /* ----------------------- Create Headers ----------------------- */
 
   const headers: { [k: string]: string } = {};
 
-  if (usingHarvardApigee) {
-    // Using Harvard Apigee system
-    headers['X-Api-Key'] = token;
-  } else {
-    // Directly contacting Zoom
-    headers.Authorization = `Bearer ${token}`;
-  }
+  // Add token to request
+  headers.Authorization = `Bearer ${token}`;
```

Comment:
Looks like no going back

Reply:
I'm redoing this to allow for rolling back